lib/resources/r4/communication.yaml

```diff
@@ -0,0 +1,193 @@
+---
+name: Communication
```

Comment:
need something for the email-status. Check out financial-transaction-method

Reply:
In my review: https://github.com/cerner/fhir.cerner.com/pull/435/files?file-filters%5B%5D=.md&file-filters%5B%5D=.yaml#diff-f08ae1e4c063b9a36a22adaa43fecf21R174

Reply:
> need something for the email-status. Check out financial-transaction-method

@VelocityKnight added this in her PR: https://github.com/cerner/fhir.cerner.com/pull/435/files#diff-f08ae1e4c063b9a36a22adaa43fecf21R174